src/patterns/atoms/forms/checkboxes.hbs

```diff
@@ -6,6 +6,8 @@ tips: |
   - Wrap the `label` element around both the control and its text label. This gives a bigger target for clicking.
   - Use the `mzp-u-inline` class to override the base block-level label styling.
   - Use a single checkbox for a binary choice (such as a yes or no question) rather than two radio buttons, especially if the field isn’t required. A checked radio button can’t be unchecked, whereas a single checkbox behaves as a switch to let a user toggle between the two states (you can uncheck a checked checkbox).
+links:
+  See the choices component for styled checkboxes: /demos/forms/choices.html
```

Comment:
We should probably link to /patterns/molecules/forms.html#custom-checkboxes-and-radio-buttons rather than the demo